tests/phpunit/integration/Core/Util/Auto_UpdatesTest.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Google_URL_Matcher_TraitTest
+ *
+ * @package   Google\Site_Kit\Tests\Core\Util
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Tests\Core\Util;
+
+use Google\Site_Kit\Core\Util\Auto_Updates;
+use Google\Site_Kit\Tests\TestCase;
+
+/**
+ * @group Util
+ */
+class Auto_UpdatesTest extends TestCase {
+	public function test_sitekit_autoupdate_forced() {
+		// Forced auto updates are available for WP >= 5.6 only.
+		if ( version_compare( get_bloginfo( 'version' ), '5.6', '<' ) ) {
+			return;
+		}
+
+		// Is null by default.
+		$this->assertNull( Auto_Updates::sitekit_forced_autoupdates_status() );
+
+		// force enable plugin autoupdates should return true
+		add_filter( 'auto_update_plugin', '__return_true' );
+
+		$this->assertTrue( Auto_Updates::sitekit_forced_autoupdates_status() );
+		$this->assertTrue( Auto_Updates::is_sitekit_autoupdates_enabled() );
+
+		// force disable plugin autoupdates should return false
+		remove_filter( 'auto_update_plugin', '__return_true' );
+		add_filter( 'auto_update_plugin', '__return_false' );
+
+		$this->assertFalse( Auto_Updates::sitekit_forced_autoupdates_status() );
+		$this->assertFalse( Auto_Updates::is_plugin_autoupdates_enabled() );
+		$this->assertFalse( Auto_Updates::is_sitekit_autoupdates_enabled() );
+
+		// cleanup
+		remove_filter( 'auto_update_plugin', '__return_false' );
```

Comment:
Can we match these to the status constants instead of their raw values? Just to improve readability a bit 🙂